### PR TITLE
Copy sig-cli gce serial tests to their own job.

### PR DESCRIPTION
**Pull request by pwittrock**
Trial balloon for splitting all kubectl tests out of shared jobs.

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -3012,6 +3012,22 @@
       "UNKNOWN"
     ]
   },
+  "sig-cli-ci-kubernetes-e2e-gce-serial": {
```
Comment (krzyzacy) [inline jobs/config.json]: The name probably should be `ci-kubernetes-e2e-gce-sig-cli-serial` or `ci-kubernetes-e2e-gce-serial-sig-cli` to be consistent with other jobs :-)

File: prow/config.yaml [hunk 2]
```diff
@@ -5799,6 +5799,69 @@ periodics:
...
+        name: ssh
+        readOnly: true
+    volumes:
+    - name: service
+      secret:
+        secretName: service-account
+    - name: ssh
+      secret:
+        defaultMode: 256
+        secretName: ssh-key-secret- interval: 30m
```
Comment (krzyzacy) [inline prow/config.yaml]: split the lines here

Comment (krzyzacy) [inline prow/config.yaml]: oh I think you copied twice :-)

**Comment by pwittrock** [comment]
@krzyzacy Comments addressed and tests are passing.  PTAL

**Comment by krzyzacy** [comment]
thanks for the project donation :-)

/lgtm

**Pull request merged**